### PR TITLE
dnn/native: empty line

### PR DESCRIPTION
**Pull request by MingyuY**
comments eg

### CODE REVIEW

**Comment by Semmer2** [comment]
eg comment for mingyu

File: libavfilter/dnn/dnn_backend_native_layer_mathunary.c [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /*
+ *
```
Comment (Semmer2) [inline libavfilter/dnn/dnn_backend_native_layer_mathunary.c:2]: Is this line useless?

File: libavfilter/dnn/dnn_backend_native_layer_mathunary.h [hunk 2]
```diff
@@ -18,6 +18,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
+
```
Comment (Semmer2) [inline libavfilter/dnn/dnn_backend_native_layer_mathunary.h:21]: Same as above.

**Comment by MingyuY** [comment]
thanks ，I change it and see what happens 


On 07/24/2020 14:59, Semmer2 wrote:

@Semmer2 commented on this pull request.

In libavfilter/dnn/dnn_backend_native_layer_mathunary.h:

> @@ -18,6 +18,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
+


Same as above.

**Pull request closed without merging**